hail/python/hail/backend/spark_backend.py

```diff
@@ -25,7 +25,7 @@
 from ..hail_logging import Logger
 
 if pyspark.__version__ < '3' and sys.version_info > (3, 8):
```

Comment:
If the `>` symbol there is to be believed should we require 3.7 or 3.8? Or should that be `>=`?

Reply:
If it's `>=` then maybe that fix is out of scope here since that would be breaking?

Reply:
Huh, this must not actually run, since we definitely allow users to use 3.8. I'll investigate, good catch.

Reply:
Oh wait, I remember now. This is because Spark 2 can't use > 3.8. So yeah, 3.7 is the only ok version, that's fine. I plan on dropping Spark 2 support shortly anyway.